openlibrary/templates/account/books.html

```diff
@@ -33,8 +40,17 @@
   $ og_description = _("%(username)s has read %(total)d books. Join %(username)s on OpenLibrary.org and tell the world about the books that you care about.", username=userDisplayName, total=total_items)
 $elif key == 'sponsorships':
   $ header_title = _("Sponsorships")
+  $ breadcrumb = header_title
   $ og_title = _("Books %(userdisplayname)s is sponsoring", userdisplayname=userDisplayName)
   $ og_description = "{username} is sponsoring {total} books. Join {username} on OpenLibrary.org and share the books that you'll soon be reading!".format(username=userDisplayName, total=total_items)
+$elif key == 'notes':
+  $ header_title = _("Book Notes")
+  $ breadcrumb = header_title
+  $ og_title = None
+$elif key == 'observations':
+  $ header_title = _("Observations")
+  $ breadcrumb = header_title
+  $ og_title = None
```

Comment:
@mekarpeles Forgot to mention: I made the assumption that open graph metadata is not needed for notes and observations pages.  Is this correct?